src/lizaalert/courses/admin.py

```diff
@@ -45,6 +47,23 @@ class ChapterInline(admin.TabularInline):
     model = Chapter
     min_num = 1
     extra = 0
+    readonly_fields = ("get_chapter_link",)
+    # Устанавливаем порядок отображения полей
+    fields = (
+        "get_chapter_link",
+        "deleted_at",
+        "order_number",
+        "title",
+        "user_created",
+        "user_modifier",
+    )
```

Comment:
поля created и updated автозаполняемые

```suggestion
    fields = (
        "get_chapter_link",
        "order_number",
        "title",
        "deleted_at",
    )
```

Reply:
Может быть fields вообще убрать?